osc-check_repo.py

```diff
@@ -36,10 +36,14 @@
 from osclib.conf import Config
 from osclib.checkrepo import CheckRepo
 from osclib.checkrepo import BINCACHE, DOWNLOADS
+from osclib.cache import Cache
 from osclib.cycle import CycleDetector
 from osclib.memoize import CACHEDIR
+from osclib.memoize import save_cache_path
 from osclib.request_finder import RequestFinder
 
+Cache.CACHE_DIR = save_cache_path('.opensuse-repo-checker-http')
```

Comment:
Either way, figured I would keep it hidden like it normally is with default configuration.